function-aws-custom-runtime/src/main/java/io/micronaut/function/aws/runtime/MicronautLambdaRuntime.java

```diff
@@ -48,17 +45,8 @@
  */
 public class MicronautLambdaRuntime {
 
-    /**
-     * @deprecated Use {@value LambdaRuntimeInvocationResponseHeaders#LAMBDA_RUNTIME_AWS_REQUEST_ID} instead
-     */
-    @Deprecated
-    public static final String HEADER_RUNTIME_AWS_REQUEST_ID = LambdaRuntimeInvocationResponseHeaders.LAMBDA_RUNTIME_AWS_REQUEST_ID;
-
-    public static final UriTemplate INVOCATION_TEMPLATE = UriTemplate.of("/2018-06-01/runtime/invocation/{requestId}/response");
-    public static final UriTemplate ERROR_TEMPLATE = UriTemplate.of("/2018-06-01/runtime/invocation/$requestId/error");
-    public static final String NEXT_INVOCATION_URI = "/2018-06-01/runtime/invocation/next";
-    public static final String INIT_ERROR_URI = "/2018-06-01/runtime/init/error";
-
+    protected AwsLambdaRuntimeApi awslambdaRuntimeApi = new AwsLambdaRuntimeApi() { };
```

Comment:
Can `MicronautLambdaRuntime` not just implement `AwsLambdaRuntimeApi` instead of creating an anonymous instance?

Reply:
done.